website/pages/docs/reference/cli/cloudquery_addon_download.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "addon_download"
+---
+## cloudquery addon download
+
+Download addon from CloudQuery Hub.
+
+### Synopsis
+
+Download addon from CloudQuery Hub.
+
+This downloads an addon from CloudQuery Hub to local disk.
```

Comment:
You could add a link (or links) so people would know where to find the addons: https://hub.cloudquery.io/addons/transformation, https://hub.cloudquery.io/addons/visualization

Reply:
@pilvikala 
There are no hub links so far in any of the cli commands, if we're going to do that maybe we should amend all our "cli frontend" with links? The only link in the cli is in the main help output, and goes like:

```
CloudQuery CLI

Open source data integration at scale.

Find more information at:
	https://www.cloudquery.io
```